src/app/components/CtaLink/index.tsx

```diff
@@ -0,0 +1,24 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import Text from '../Text';
+import styles from './index.styles';
+
+interface CtaLinkProps {
+  href?: string;
+  className?: string;
+  children?: React.ReactNode;
+}
+
+const CtaLink = ({ href, className, children }: CtaLinkProps) => {
```

Comment:
Nit picky, but you can use Reacts own `PropsWithChildren<CtaLinkProps>` here which means you don't have to define `children?:...` above.